csharp/src/Ice/OpaqueEndpointI.cs

```diff
@@ -59,12 +59,12 @@ public override void StreamWriteImpl(Ice.OutputStream s)
         public override string ToString()
         {
             string val = System.Convert.ToBase64String(_rawBytes);
-            return "opaque -t " + _type + " -e " + Ice.Util.EncodingVersionToString(_rawEncoding) + " -v " + val;
+            return "opaque -t " + _type + " -e " + Ice.Util.EncodingToString(_rawEncoding) + " -v " + val;
```

Comment:
I think we should move the ToString to the Encoding and Protocol classes

Reply:
Agreed, but for a separate PR.